caffe2/python/onnx/frontend.py

```diff
@@ -510,10 +544,18 @@ def ssa_name(name, version):
             return '{}_{}'.format(name, version)
 
         if init_net:
-            for op in init_net.op:
-                assert re.match('GivenTensor.*Fill', op.type)
+            index = 0
+            while index < len(init_net.op):
+                op = init_net.op[index]
                 assert len(op.output) == 1
+                if op.output[0] in value_info:
```

Comment:
It's safer to whitelist couple operators types that can be deleted.

Reply:
ConstantFill and Given*TensorFill